docs/strict_mock/index.rst

```diff
@@ -542,8 +542,32 @@ If this type validation yields bad results (likely a bug, please report it), you
 
   StrictMock(template=SomeClass, type_validation=False)
 
+Attribute Type Validation
+=========================
+By default, ``StrictMock`` will validate types of set attributes, method call arguments and method return values, against available type hinting information.
+If you want to disable type validation for certain attributes pass ``attributes_to_skip_type_validation`` to the constructor of ``StrictMock``
```

Comment:
`type_validation=False` works in combination with `attributes_to_skip_type_validation`: either global or per attribute. Perhaps it is worth explaining this, maybe merge this section with the previous section (they're kind of the same).

Reply:
Sorry I am bad at writing docs :D